rosdep/python.yaml

```diff
@@ -4950,6 +4953,7 @@ python-virtualenv:
   fedora: [python-virtualenv]
   gentoo: [dev-python/virtualenv]
   rhel:
+    '*': [python2-virtualenv]
```

Comment:
python2-virtualenv in CentOS 8 http://rpmfind.net/linux/RPM/centos/8.1.1911/appstream/aarch64/Packages/python2-virtualenv-15.1.0-19.module_el8.1.0+219+cf9e6ac9.noarch.html